modules/psap-node-feature-discovery-topology-updater-command-reference.adoc

```diff
@@ -0,0 +1,190 @@
+== -version
+
+Print version and exit.
+
+[discrete]
+[id="nfd-topology-updater-server_{context}"]
+== -server
+
+The `-server` flag specifies the address of the nfd-master endpoint where to connect to.
+
+Default: `localhost:8080`
+
+.Example
+[source,terminal]
+----
+$ nfd-topology-updater -server=nfd-master.nfd.svc.cluster.local:443
+----
+
+[discrete]
+[id="nfd-topology-updater-ca-file_{context}"]
+== -ca-file
+
+The `-ca-file` flag is one of the three flags, together with `-cert-file` and `-key-file`, controlling the mutual TLS authentication on the NFD Topology Updater. This flag specifies the TLS root certificate that is used for verifying the authenticity of nfd-master.
+
+Default: empty
+
+[IMPORTANT]
+====
+The `-ca-file` flag must be specified together with `-cert-file` and `-key-file` flags.
+====
+
+.Example
+[source,terminal]
+----
+$ nfd-topology-updater -ca-file=/opt/nfd/ca.crt -cert-file=/opt/nfd/updater.crt -key-file=/opt/nfd/updater.key
+----
+
+[discrete]
+[id="nfd-topology-updater-cert-file_{context}"]
+== -cert-file
+
+The `-cert-file` flag is one of the three flags, together with `-ca-file` and `-key-file`, controlling mutual TLS authentication on the NFD Topology Updater. This flag specifies the TLS certificate presented for authenticating outgoing requests.
+
+Default: empty
+
+[IMPORTANT]
+====
+The `-cert-file` flag must be specified together with the `-ca-file` and `-key-file` flags.
+====
+
+.Example
+[source,terminal]
+----
+$ nfd-topology-updater -cert-file=/opt/nfd/updater.crt -key-file=/opt/nfd/updater.key -ca-file=/opt/nfd/ca.crt
+----
+
+[discrete]
+[id="nfd-topology-updater-key-file_{context}"]
+== -key-file
+
+The `-key-file` flag is one of the three flags, together with `-ca-file` and `-cert-file`, controlling the mutual TLS authentication on the NFD Topology Updater. This flag specifies the private key corresponding the given certificate file, or `-cert-file`, that is used for authenticating outgoing requests.
+
+Default: empty
+
+[IMPORTANT]
+====
+The `-key-file` flag must be specified together with the `-ca-file` and `-cert-file` flags.
+====
+
+.Example
+[source,terminal]
+----
+$ nfd-topology-updater -key-file=/opt/nfd/updater.key -cert-file=/opt/nfd/updater.crt -ca-file=/opt/nfd/ca.crt
+----
+
+[discrete]
+[id="nfd-topology-updater-server-name-override_{context}"]
+== -server-name-override
+
+The `-server-name-override` flag specifies the common name (CN) which to expect from the nfd-master TLS certificate. This flag is mostly intended for development and debugging purposes.
+
+Default: empty
+
+.Example
+[source,terminal]
+----
+$ nfd-topology-updater -server-name-override=localhost
+----
+
+[discrete]
+[id="nfd-topology-updater-no-publish_{context}"]
+== -no-publish
+
+The `-no-publish` flag disables all communication with the nfd-master, making it a "dry-run" flag for nfd-topology-updater. NFD Topology Updater resource hardware topology detection normally, but no CR requests are sent to nfd-master.
```

Comment:
* Does "dry-run" really have to be in quotes, or can they be removed?
* I don't understand what this part of the sentence is saying "NFD Topology Updater resource hardware topology detection normally", is there a word missing or something?